src/components/Table/Table.stories.tsx

```diff
@@ -19,6 +19,7 @@
 import type { Meta, StoryObj } from '@storybook/react'
 import { action } from '@storybook/addon-actions'
 
+import './style.css'
```

Comment:
non importiamo così gli stili poiché possono cambiare altri stili quando il componente viene importato.
usiamo i css module in modo che ogni classe sia poi interpolata attraverso un hash rendendola praticamente univoca.

Reply:
Risolto